### PR TITLE
Sync torpedo to sched-ops api split changes

### PR DESCRIPTION
**Pull request by raviprasadsn**
Added changes to reflect API split in schedops.

### CODE REVIEW

**Comment by harsh-px** [comment]
The CI needs to be fixed. Looks like a vendoring issue.

**Comment by disrani-px** [comment]
@raviprasadsn Please put the vendor changes in a separate commit

**Comment by harsh-px** [comment]
The DeleteTasks API is failing on this branch with the below from the API server.

```
found '<', expected: !, identifier, or 'end of string'
```

We are working on root causing this.

File: drivers/scheduler/k8s/k8s.go [hunk 1]
```diff
@@ -18,7 +18,15 @@ import (
...
-	k8sops "github.com/portworx/sched-ops/k8s"
+	"github.com/portworx/sched-ops/k8s/apps"
+	"github.com/portworx/sched-ops/k8s/autopilot"
+	k8sCommon "github.com/portworx/sched-ops/k8s/common"
+	"github.com/portworx/sched-ops/k8s/core"
+	"github.com/portworx/sched-ops/k8s/externalstorage"
+	"github.com/portworx/sched-ops/k8s/rbac"
+	"github.com/portworx/sched-ops/k8s/storage"
+	"github.com/portworx/sched-ops/k8s/stork"
+
```
Comment (harsh-px) [inline drivers/scheduler/k8s/k8s.go]: remove extra line

**Pull request merged**